.github/workflows/ci-kafka.yml

```diff
@@ -18,6 +18,10 @@ permissions:  # added using https://github.com/step-security/secure-workflows
 jobs:
   kafka:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        name: [v1, v2]
```

Comment:
will `name` be also the name of the job? Or do you need to add a `name: ` directive using this var?

Reply:
nit: I would use `version` instead of `name` if it's just a var

Reply:
> nit: I would use `version` instead of `name` if it's just a var

Yes, it is just a var. I'll change that.